src/components/EnvironmentVariables/index.js

```diff
@@ -0,0 +1,427 @@
+        varTarget="Environment"
+      />
+      <div className="field-wrapper env-vars">
+        {environment.envVariables.length == 0 ? (
+          <>
+            <hr style={{ margin: "30px 0" }} />
+            <div style={{ textAlign: "center" }}>
+              No Environment variable set
+            </div>
+          </>
+        ) : (
+          <>
+            <div className="header">
+              <label>Environment Variables</label>
+              <Button
+                onClick={() => showVarValue()}
+                aria-controls="example-collapse-text"
+                aria-expanded={openEnvVars}
+              >
+                {!openEnvVars ? "Show values" : "Hide values"}
+              </Button>
+            </div>
+            <Table striped bordered hover>
+              <thead>
+                <tr>
+                  <th>Environment Variable Name</th>
+                  <th>Environment Variable Scope</th>
+                  <Collapse in={openEnvVars}>
+                    <th>Value</th>
+                  </Collapse>
+                  <th>Delete</th>
+                </tr>
+              </thead>
+              <tbody>
+                {displayVars.map((envVar, index) => {
+                  return (
+                    <Fragment key={index}>
+                      <tr>
+                        <td className="varName">{envVar.name}</td>
+                        <td className="varScope">{envVar.scope}</td>
+                        {envLoading ? (
+                          <Collapse in={openEnvVars}>
+                            <td className="varValue" id={index}>
+                              <div className="loader"></div>
+                            </td>
+                          </Collapse>
+                        ) : envVar.value ? (
+                          <Collapse in={openEnvVars}>
+                            <td className="varValue" id={index}>
+                              {envVar.value.length <= 100 &&
+                              !valueState[index] ? (
+                                <div className="showHideContainer">
+                                  {hashValue(envVar.value).substring(0, 25)}
+                                  <a href="#" onClick={() => valuesShow(index)}>
+                                    <Image
+                                      src={show}
+                                      className="showHide"
+                                      style={{ all: "unset" }}
+                                      alt=""
+                                    />
+                                  </a>
+                                </div>
+                              ) : envVar.value.length <= 100 &&
+                                valueState[index] ? (
+                                <div className="showHideContainer">
+                                  {envVar.value}
+                                  <a href="#" onClick={() => valuesHide(index)}>
```

Comment:
better to use `span` here since we're actually not navigating anywhere, or an inline-block div